edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/DispatcherTest.cs

```diff
@@ -20,9 +20,9 @@ namespace Microsoft.Azure.Devices.Routing.Core.Test
     [ExcludeFromCodeCoverage]
```

Comment:
Please help to remove it, otherwise it doesn't include in test coverage calculation.

Reply:
for sure